Main/FileListItem.h

```diff
@@ -31,6 +32,8 @@ friend	void PecoApp::CreateScript(BMessage *msg);
 	void			SetNewName( BString	myNewName );
 	void 			SetName( BString name );
 	bool			CompareWith(FileListItem *CompareItem);
+	void			SetRow(FileListRow* row) { fRow = row; }
+	FileListRow*	GetRow() const { return fRow; }
```

Comment:
No. I followed the indentation of the original code (tab stop = 4). Here it is displayed with tab stop = 8.